Source/ACE/Entity/Player.cs

```diff
@@ -215,16 +216,16 @@ public Player(Session session, AceCharacter character) : base(character)
 
             // FIXME(ddevec): Once physics data is refactored this shouldn't be needed
             SetPhysicsState(PhysicsState.IgnoreCollision | PhysicsState.Gravity | PhysicsState.Hidden | PhysicsState.EdgeSlide, false);
-            PhysicsData.PhysicsDescriptionFlag = PhysicsDescriptionFlag.CSetup | PhysicsDescriptionFlag.MTable | PhysicsDescriptionFlag.Stable | PhysicsDescriptionFlag.Petable | PhysicsDescriptionFlag.Position | PhysicsDescriptionFlag.Movement;
+            PhysicsDescriptionFlag = PhysicsDescriptionFlag.CSetup | PhysicsDescriptionFlag.MTable | PhysicsDescriptionFlag.Stable | PhysicsDescriptionFlag.Petable | PhysicsDescriptionFlag.Position | PhysicsDescriptionFlag.Movement;
 
             // apply defaults.  "Load" should be overwriting these with values specific to the character
             // TODO: Load from database should be loading player data - including inventroy and positions
-            PhysicsData.CurrentMotionState = new UniversalMotion(MotionStance.Standing);
+            CurrentMotionState = new UniversalMotion(MotionStance.Standing);
 
-            PhysicsData.MTableResourceId = 0x09000001u;
-            PhysicsData.Stable = 0x20000001u;
-            PhysicsData.Petable = 0x34000004u;
-            PhysicsData.CSetup = 0x02000001u;
+            MotionTableId = 0x09000001u;
```

Comment:
They should be fine to remove. I did not remove them yet because I have done little to no work on player and did not really have time to test. I would really hate to hold this up for that. How about a big todo and when I have more time, I will clean it up?